OasisDEX/oasis-borrow
# Automation links update

# [Automation links update](https://app.shortcut.com/oazo-apps/story/8279/update-automation-kb-links)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- updated links
  
## How to test 🧪
  <Please explain how to test your changes>

- check places from story to verify whether links has been updated
